guides/routing.md

```diff
@@ -98,7 +98,7 @@ DELETE  /users/:id       HelloWeb.UserController :delete
 ...
 ```
 
-This is the standard matrix of HTTP verbs, paths, and controller actions. For a while, this was known as RESTful routes, but most consider this a misnomer nowadays. Let's look at them individually, in a slightly different order.
+This is the standard matrix of HTTP verbs, paths, and controller actions. For a while, this was known as RESTful routes, but most consider this a misnomer nowadays. Let's look at them individually in the same order.
```

Comment:
Personally, I would just omit anything about "order" here since it doesn't add much and is brittle anytime docs are updated.

> Let's look at them individually:

Reply:
Agreed.